zebra/rt.h

```diff
@@ -31,6 +31,9 @@
 #include "zebra/zebra_mpls.h"
 #include "zebra/zebra_dplane.h"
 
+#define RSYSTEM_ROUTE(type)                                                    \
+	((type) == ZEBRA_ROUTE_KERNEL || (type) == ZEBRA_ROUTE_CONNECT)
+
```

Comment:
very useful to have a combo macro like this - but we've also got RIB_SYSTEM_ROUTE in zebra/zebra_rib.c. it'd be clearer to have just one flavor?